extensions/wikia/TrackingOptIn/styles/ext.wikia.trackingSettingsManager.css

```diff
@@ -0,0 +1,7 @@
+#privacy-settings-button {
+    cursor: pointer;
```

Comment:
not needed - it's already in `button.wds-button`

Reply:
by default it's not a pointer